elements/urth-core-dataframe/urth-core-dataframe.html

```diff
@@ -272,9 +282,30 @@
          */
         onModelValueChange: function(newVal){
             this._debug( "urth-core-dataframe onModelValueChange", newVal );
+            newVal = this._deserializeDateTypes(newVal || {data:[]});
             this._setValue(newVal || {data:[]});
         },
 
+        /*
+         * Returns the dataFrame with deserialized Date types of the dataFrame reconstructed via the column_types
+         * @param df - the serialized DataFrame
+         * @return the reconstructed DataFrame
+         */
+        _deserializeDateTypes: function(df) {
+            if (df.column_types == null || df.data == null) {
```

Comment:
Being picky here, but in JavaScript, we don't really check for null when things are undefined. The if statement above is better is we do `if(!df.column_types || !df.data)`